setup.cfg

```diff
@@ -1,4 +1,5 @@
 [flake8]
 ignore = E501
 exclude = node_modules/*
-max-complexity = 10
\ No newline at end of file
+max-complexity = 10
+per-file-ignores = stopcovid/dialog/dialog.py:E721
```

Comment:
flake8 is confused by the event_from_dict method. it thinks we're doing actual type checks and should be using isinstance